Makefile

```
@@ -25,6 +25,10 @@
 .PHONY: container-images publish-container-images validate-runtime-images
 SHELL:=/bin/bash
 
+# Python execs
+PYTHON=python3
```

I think we should allow for overrides on the CLI...
```suggestion
PYTHON?=python3
```